plugins/inventory-resources/src/components/EditProduct.svelte

```diff
@@ -0,0 +1,152 @@
+<!--
```

Comment:
I think we need some unified editor component to be used with EditContact, EditProduct.